docs/process/release.md

```diff
@@ -101,6 +101,26 @@ git tag 'vXX'
 git push upstream && git push upstream --tags
 ```
 
+### Authenticating
+
+**This can ONLY be done by a Google employee.**
+
+Log in to [NPM](https://npmjs.com/) to the `angular` account in order to
+publish. This account is protected by two factor authentication (2FA).
+
+Run `npm login`:
+
+1. For username, use `angular`.
+1. For password, use the value stored [here](http://go/ng-npm-pass).
+1. For email, use `devops+npm@angular.io`.
+1. For two-factor code, you'll need to set up add an account.
```

Comment:
set up an account?

Reply:
Good catch, changed to `set up the account`.